### ISSUE TITLE
Releases are broken due to dottydoc

### ISSUE THREAD

**Issue by olafurpg** [scalameta/munit#210]
From https://github.com/scalameta/munit/runs/1192541140#step:6:496

```
Error:  java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
Error:  	at xsbt.DottydocRunner.run(DottydocRunner.java:85)
Error:  	at xsbt.ScaladocInterface.run(ScaladocInterface.java:11)
```

cc/ @tgodzik @OlivierBlanvillain any suggestions on how to fix this problem?

**Comment by tgodzik** [comment]
@smarter noticed it a while ago:
```
FYI, munit doc publishing is broken (https://github.com/scalameta/munit/runs/1118544661), because you switched to sbt 1.4.0-RC1 and dottydoc is doing something dumb, I have a PR for that: lampepfl/dotty#9756
meanwhile, you may be able to turn off this stuff by doing something like sources in (Compile, doc) := if (isDotty.value) Seq() else (sources in (Compile, doc)).value
```
since that is merged I was hoping we can just update at some point, but I guess that would get updated with a dotty version. So we might need to do the workaround for now.

**Comment by olafurpg** [comment]
I pushed a commit downgrading sbt to 1.3.13, let's see if that fixes the issue 565e21ac74084679d767385f3c5561292ea77268

**Comment by olafurpg** [comment]
Looks like that did the trick, thank you @tgodzik !
